axum/Cargo.toml

```diff
@@ -50,6 +50,10 @@ tower = { version = "0.4.13", default-features = false, features = ["util"] }
 tower-layer = "0.3.2"
 tower-service = "0.3"
 
+# wont need this when axum uses http-body 1.0
+hyper1 = { package = "hyper", version = "1.0.0-rc.3", features = ["server", "http1"] }
+tower-hyper-http-body-compat = { version = "0.1.3", features = ["server", "http1"] }
```

Comment:
Just a stepping stone until we've ported axum to hyper 1.0. We won't actually ship 0.7 with this.